nitraa-pune/routes/userSignin/index.js

```diff
@@ -37,34 +37,34 @@ router.post('/', function (req, res, next) {
                     } else {
                         db.close();
                         var jwt_token = jwt.sign({
-                            db_id: result._id
+                            usermail: req.body.email
```

Comment:
change the `usermail: req.body.email` to `tid: result._id`

Reply:
Required changes done